.kokoro/static_analysis/common.cfg

```diff
@@ -19,13 +19,19 @@ gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/trampoline"
 build_file: "java-docs-samples/.kokoro/trampoline.sh"
 env_vars: {
     key: "TRAMPOLINE_IMAGE"
-    value: "gcr.io/cloud-devrel-kokoro-resources/java8"
+    value: "gcr.io/cloud-devrel-kokoro-resources/java11"
 }
 env_vars: {
     key: "TRAMPOLINE_BUILD_FILE"
     value: "github/java-docs-samples/.kokoro/tests/run_static_analysis.sh"
 }
 
+# Set the JAVA VERSION env var.
+env_vars: {
+    key: "JAVA_VERSION"
+    value: "1.8,11"
+}
+
```

Comment:
The static analysis check doesn't use this: https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/.kokoro/tests/run_static_analysis.sh

Reply:
Maven uses this though doesn't it?

Reply:
Or I guess static analysis doesn't run any mvn commands.

Reply:
Maven doesn't, there is something in the other script we used to control which versions are run:
https://github.com/GoogleCloudPlatform/java-docs-samples/blob/a683a732cff3d6833815461c2d59bf3f64c5c0a6/.kokoro/tests/run_tests.sh#L159-L164